openapi/zuora.revenue/openapi.yml

```diff
@@ -0,0 +1,1486 @@
+      target="_blank">Integration service (BI Views V2)</a> in Zuora Revenue
+      Knowledge Center.
+  - name: File Upload
+    description: >
+      The File Upload operations are used to upload large volumes of data as a
+      CSV file in the HTTP body to Zuora Revenue and to get status information
+      about the file upload.
+
+
+      If the data volume is in the range of 40k-60k records per file per
+      request, the Upload file operation can process  the data quickly. 
+
+      A unique request ID is generated after the file is successfully received,
+      which can be used  to query the upload status in the Get file upload
+      status operation.    
+  - name: Inbound
+    description: >
+      The Inbound operations are used to upload data in the CSV format to Zuora
+      Revenue. As a prerequisite,  the upload template definitions must be
+      present in Zuora Revenue. Then, you specify the template name,  file name,
+      authentication token, and the CSV data in HTTP body of the inbound
+      operations to upload data. It is recommended to use the Create upload
+      operation for the CSV file than contains less than 20k lines. 
+
+
+      After the data is uploaded to the pre-stage table, Zuora Revenue scheduler
+      will load the CSV content to corresponding staging tables in Zuora
+      Revenue. You can query the upload status by using the Get upload status
+      operation.
+  - name: Reports
+    description: >
+      The Reports operations are used to download reports that are generated in
+      Zuora Revenue. 
+  - name: Revenue Jobs
+    description: >
+      The Revenue Jobs operations are used to submit the data collection jobs
+      and query the status of the submitted jobs.
+  - name: Stage Error
+    description: >
+      The Stage Error operation is used to get the list of records that contain
+      errors in the Zuora Revenue staging tables for transactions or events.
+  - name: Transfer Accounting
+    description: >
+      The Transfer Accounting operations are used to support outbound data
+      integrations. You  can use these operations to query the transfer batch
+      information and to update the transfer batch status.
+  - name: Transfer Accounting Files
+    description: >
+      The Transfer Accounting Files operation is used to download the transfer
+      accounting  data in a CSV file from Zuora Revenue for a specific transfer
+      batch.
+paths:
+  /api/integration/v1/fileupload/status/{file_request_id}:
+    get:
+      tags:
+        - File Upload
+      summary: Get file upload status
+      description: Gets the status information about the specified file upload request.
+      operationId: GET_FileUploadStatus
+      parameters:
+        - name: file_request_id
+          in: path
+          description: The request ID of the file upload to be queried.
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
```

Comment:
We have an open [issue in openapi-tool ](https://github.com/ballerina-platform/openapi-tools/issues/696)side to address this scenario where either of `200` and `204` responses are expected. Current code generation does not support this scenario.